sway/swaynag.c

```diff
@@ -0,0 +1,103 @@
+#include <fcntl.h>
+#include <signal.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include "log.h"
+#include "sway/swaynag.h"
+
+void swaynag_clone(struct swaynag_instance *dest,
+		struct swaynag_instance *src) {
+	dest->args = src->args;
+	dest->pid = src->pid;
+	dest->fd[0] = src->fd[0];
+	dest->fd[1] = src->fd[1];
+	dest->detailed = src->detailed;
+}
```

Comment:
Why does this exist when memcpy exists

Reply:
I'm not sure what was going through my head when I did that. Switched to using `memcpy`.